src/frontend/src/tests/pages/ProjectDetailPage/TaskList.test.tsx

```diff
@@ -21,7 +21,7 @@ const renderComponent = () => {
   const RouterWrapper = routerWrapperBuilder({});
   return render(
     <RouterWrapper>
-      <TaskList />
+      <TaskList tasks={[]} />
```

Comment:
I generally like having *some* tests, even if only a snapshot test or something. Like show the table heading text is there, or show that it displays the "no tasks" thing shows up properly. If you don't have time it would be nice to spin off a ticket